pyracf/common/invalid_segment_trait_error.py

```diff
@@ -0,0 +1,18 @@
+"""Exception to use when the user passes invalid segment-trait(s) in the traits dictionary."""
```

Comment:
I suggest making the same changes that I suggested for `InvalidSegmentNameError` here too.

Reply:
changed to:
`"""Exception to use when the user passes bad segment-trait name(s) on an add/alter request."""`